deployment/v3/utils/prop_migrator/knowledge/new-property-with-decent-default-value.csv

```diff
@@ -1406,14 +1274,13 @@ resident-default.properties,sequence-order
 resident-default.properties,resident.template.bell-icon.request-received.order-a-physical-card
 resident-default.properties,mosip.resident.sign.pdf.reference.id
 resident-default.properties,resident.template.ack.update-demographic-data
-resident-default.properties,resident.update-uin.machine-name-prefix 
+resident-default.properties,resident.update-uin.machine-name-prefix
 resident-default.properties,mosip.scope.resident.postAuthTypeStatus
 resident-default.properties,resident.template.tnc.share-cred-with-partner
 resident-default.properties,mosip.resident.ack.personalised_card.name.convention
 resident-default.properties,resident.template.bell-icon.success.update-demo-data
 resident-default.properties,resident.template.bell-icon.request-received.verify-my-phone-email
 resident-default.properties,resident.template.email.subject.success.vid-card-download
-resident-default.properties,otpChannel.mobile
```

Comment:
this should have only 133 lines removed

Reply:
I have checked the count, the number of properties present now is 1625 and previously we had 1758. so 1758-1625=133 the properties removed is accurate sir.